src/tests/Uns.test.ts

```diff
@@ -626,6 +626,21 @@ describe('UNS', () => {
     });
   });
 
+  describe('#owner', () => {
+    it('should not throw when resolver is null', async () => {
+      const spies = mockAsyncMethods(uns, {
+        get: {
+          owner: '0x58cA45E932a88b2E7D0130712B3AA9fB7c5781e2',
+          resolver: null,
+          records: {},
+        },
+      });
+      const owner = await uns.owner('ryan.crypto');
```

Comment:
@sudoryan This test will be false-positive in live since your's domain has resolver. 

Use telekinetic.crypto instead